gr-uhd/lib/usrp_sink_impl.cc

```diff
@@ -400,7 +400,7 @@ int usrp_sink_impl::work(int noutput_items,
             // There is a tag gap since no length_tag was found immediately following
             // the last sample of the previous burst. Drop samples until the next
             // length_tag is found. Notify the user of the tag gap.
-            std::cerr << "tG" << std::flush;
+            GR_LOG_ERROR(d_logger, "tG");
```

Comment:
@mbr0wn I'll need your feedback on this.

Reply:
I wonder if there's a way to force the LOG to flush ... that would then be the equivalent of the prior version, just updated for gr::logging.

Reply:
true; this isn't being flushed.

Also, now having not slept about it: it's highly undesirable to have "tG" messages flood log files, isn't it? How is UHD's logging handling that (the O's and U's and L's)?

Reply:
Pretty sure UHD just floods as necessary. It can be quite a PITA sometimes, actually.